09_dt_rtc/i2c_detection.log

```diff
@@ -0,0 +1,19 @@
+/ # 
+/ # i2cdetect -yr 2
+     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
+00:          -- -- -- -- -- -- -- -- -- -- -- -- -- 
+10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
+20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
+30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
+40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
+50: -- -- -- -- -- -- -- UU -- -- -- -- -- -- -- -- 
+60: -- -- -- -- -- -- -- -- 68 -- -- -- -- -- -- -- 
+70: -- -- -- -- -- -- -- --                         
+/ # 
+/ # i2cdump -y -r 00h-06h 2 68 
+i2cdump: invalid range
+/ # i2cdump -y -r 00-06 2 68 
+     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f    0123456789abcdef
+00: 42 37 14 05 18 03 21                               B7????!         
+/ # 
+/ # 
```

Comment:
From the task description:
---
* Using i2ctools (i2cdetect, i2cdump, etc.) read *and write* date/time registers 00h-06h, *start and stop clock* using EOSC bit in register 0Eh.

Reply:
Done.